queue/queue.py

```diff
@@ -3,14 +3,16 @@ def __init__(self):
     self.size = 0
     # what data structure should we
     # use to store queue elements?
-    
-    self.storage = 
+    self.storage = []
 
   def enqueue(self, item):
-    pass
+    self.storage.insert(0, item)
   
   def dequeue(self):
-    pass
+    if self.len() == 0:
+      return None
+    else:
+      return self.storage.pop()
 
   def len(self):
-    pass
+    return len(self.storage)
```

Comment:
While the len() function works, it is very inefficient to call this method, which has a runtime of O(n), every time you need the size of the queue.  Try to keep track of the size of the queue using self.size, a provided attribute of Queue, every time you enqueue and dequeue.